### PR TITLE
perf(docs): refactor example scripts to new system

### PR DESCRIPTION
**Pull request by jason-capsule42**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Summary:

Update JS examples to new system
## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [x] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: demo/api.html [hunk 1]
```diff
@@ -17,19 +17,19 @@
...
+
+    <!-- legacy reference is still needed to support auro-accordion's use of legacy toke vales at this time -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@alaskaairux/design-tokens@latest/dist/tokens/CSSCustomProperties.css" />
+
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@aurodesignsystem/design-tokens@latest/dist/tokens/CSSCustomProperties.css" />
     <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/@aurodesignsystem/webcorestylesheets@latest/dist/demoWrapper.css" />
     <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/@aurodesignsystem/webcorestylesheets@latest/dist/elementDemoStyles.css" />
   </head>
-  <body class="auro-markdown">
+  <body>
```
Comment (blackfalcon) [inline demo/api.html:29]:
```suggestion
  <body class="auro-markdown">
```
The class needs to be there or the page styles will not be applied.

File: demo/indexCopy.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  See LICENSE in the project root for license information.
+
+  HTML in this document is standardized and NOT to be edited.
+  All demo code should be added/edited in ./demo/apiExamples.md
+
+  With the exception of adding custom elements if needed for the demo.
+
+  ----------------------- DO NOT EDIT -----------------------------
+
+-->
```
Comment (blackfalcon) [inline demo/indexCopy.html:12]: What is this file for? `indexCopy.html`?

File: docs/partials/api.md [hunk 3]
```diff
@@ -5,10 +5,10 @@
...
 ### Default
 
-  <div class="exampleWrapper">
-    <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/default.html) -->
-    <!-- AURO-GENERATED-CONTENT:END -->
-  </div>
+<div class="exampleWrapper">
+  <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/default.html) -->
+  <!-- AURO-GENERATED-CONTENT:END -->
+</div>
```
Comment (blackfalcon) [inline docs/partials/api.md:11]: Isn't the standard for this file to be maned `apiExamples.md`? 

I see that @jordanjones243 changed this to `api.md` in this commit. Shouldn't we be updating to meet the Generator standard? 

- https://github.com/AlaskaAirlines/auro-carousel/commit/16d883dd389146b28d228c416a3f7e73c998c92c

**Review by blackfalcon — CHANGES_REQUESTED** [review]
Recommend that the merge-down is removed from the commits and merge conflicts need to be addressed before merging is allowed.

File: docs/partials/api.md (hunk 3 above)
Comment (jordanjones243) [reply docs/partials/api.md:11]: @blackfalcon The Generator was recently updated to reflect the `index.md` and `api.md` demo file naming standard

Comment (blackfalcon) [reply docs/partials/api.md:11]: This example references has been causing a lot of confusion. Please see this Discussion as we must manage a strategy for updating all the repos once more to fit this change. 

https://github.com/AlaskaAirlines/WC-Generator/discussions/522

**Comment by blackfalcon** [comment]
@jason-capsule42 the offending merge-down was removed from this PR. Please rebase this feature branch from the main branch and resolve the listed conflicts.

**Pull request closed without merging**